app/Resources/views/banner/macron.html.twig

```diff
@@ -0,0 +1,14 @@
+<img src="{{ asset('images/jevotemacron.png') }}"
+     title="Je Vote Macron"
+     alt="Je Vote Macron" />
+<div class="macron__banner__links">
+    <a href="/article/10-raisons-de-voter-emmanuel-macron">
```

Comment:
Yes. :/ I'm not sure what else I can do. It would be great to add a back-end ultimately for these links. What would you recommend?